### DISCUSSION TITLE
Article knowledge level

### DISCUSSION THREAD

**Discussion by jindradev** [dailydotdev/daily#692] [Feature Requests]
Hi, this is a stretch but could it be possible to separate the articles by knowledge level? I'm pretty tired of clicking at enticing articles about Javascript patterns or similar just to find out some guy on dev.to with 3 months of experience is trying to teach me how to write a cycle? (mostly with a wrong conclusion as well...)

Thank you. I'm just tired and overwhelmed with information :) .

**Comment by idoshamun** [comment]
Hi @jindradev,

This is a great suggestion but one of the hardest things to pull off. We would like to explore the possibility of tagging articles according to seniority/experience. In the meantime, may I suggest blocking sources that you don't like?

**Reply by lukejans** [reply]
was just going to come here for the same recommendation! an example of a platform that has this feat is https://www.geeksforgeeks.org/ 

I understand the complexity of implementing this but a couple ideas for a dev.to situation would be to filter maybe by likes and reactions on the post as well as how long the account has been active for to build a form of trust score. Obviously this wouldn't tell the difference between beginner articles and advanced ones so you could maybe add a layer to run some sort of program to filter the text for keywords like beginner or what not. Just a thought because this would be a great feature to have that would both help experienced to not consume unnecessary media as well as new devs getting the right entry they need.
